addon/modifiers/scroll-into-view.js

```diff
@@ -18,23 +18,37 @@ export default modifier(function scrollIntoView(
       ) {
         element.scrollIntoView(options);
       } else {
-        const { behavior = 'auto', topOffset, leftOffset } = options;
+        const {
+          behavior = 'auto',
+          topOffset,
+          leftOffset,
+          scrollContainerId,
+        } = options;
 
-        const left =
-          leftOffset === undefined
-            ? 0
-            : element.getBoundingClientRect().left -
-              document.body.getBoundingClientRect().left -
-              leftOffset;
+        let scrollContainer, left, top;
+        if (scrollContainerId !== undefined) {
+          scrollContainer = document.getElementById(scrollContainerId);
```

Comment:
An alternative here could be to figure out the closest scroll container relative to the element the modifier is placed on so the explicit id doesn't have to be passed.

The question however is if that kind of logic is wanted in this modifier.

Reply:
I'm not opposed to it, it just depends on whether there's a use-case for it. If we find that people are only ever passing in the id for the closest scroll container, then what you're suggesting would make sense. Otherwise, it's probably unnecessary added complexity.

Reply:
Not sure if there's an easy way to figure out the closest scroll container.

Making the reference explicit via id is _probably_ less error prone for now 🤔